docs/usage/dom-operations/css-class-mutations.md

```diff
@@ -11,6 +11,15 @@ cable_ready["MyChannel"].add_css_class(
 )
 ```
 
+Use an array to sets multiple classes to an element.
+
+```ruby
+cable_ready["MyChannel"].add_css_class(
+  selector: "string", # required - string containing a CSS selector or XPath expression
+  name:     ["string", "string2"]  # [null]   - the CSS class to add
```

Comment:
I'd update the code comment to, `the CSS class name(s) to add`, given that the value can be a `String` or an `Array`